### DISCUSSION TITLE
New install - Search node failing to show up in grid - CA errors?

### DISCUSSION THREAD

**Discussion by smitty-exos** [Security-Onion-Solutions/securityonion#10628] [2.4]
Hey all,

Fresh 2.4.2 build on Rocky 9 and pulling via git. Everything installed fine it seemed, but I notice the following in /opt/so/log/salt/minion
on the search node I just added.

2023-06-21 16:30:44,541 [salt.state       :325 ][ERROR   ][36513] ca_server did not respond. Salt master must permit peers to call the sign_remote_certificate function.
2023-06-21 16:31:23,566 [salt.state       :325 ][ERROR   ][36513] ca_server did not respond. Salt master must permit peers to call the sign_remote_certificate function.

Any help would be appreciated.

**Comment by dougburks** [comment]
Did you configure the manager to allow the search node to connect as described at https://docs.securityonion.net/en/2.4/configuration.html#production-server-distributed-deployment?

**Reply by smitty-exos** [reply]
I did....I went with the following : 
`The second option is to wait until the node tries to join and it will prompt you to run a specific command on the manager. Regardless of which of the two options you choose, it will eventually prompt you to go to [Administration](https://docs.securityonion.net/en/2.4/administration.html#administration) –> Grid Members, find the node in the Pending Members list, click the Review button, and then click the Accept button.`

The node shows up in Administration -> Grid Members

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/35970582/3a30b872-5132-41e5-ab34-a8808ce12cb6)

Does not show up in Grid

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/35970582/66ad2868-ccc7-493d-aeb2-dc712ea0ef3d)

**Reply by dougburks** [reply]
Have you checked the salt logs on the manager?

Are there any firewalls or other network devices that could be interfering with the search node connecting to the manager?
